### DISCUSSION TITLE
Detached Content support

### DISCUSSION THREAD

**Discussion by fuson** [panva/jose#362] [Q&A] [answered]
JWS may work without payload (RFC7515 Appendix F), but i can't find full support of it native. 

1. Need a possibility to generate compact JWS in detached mode

**Comment by panva** [comment, marked as answer]
The exact appendix you mention describes how to detach and re-attach the payload using the different serializations, all of which are supported. There's nothing needed from the library, which is also what said appendix says at its very end.
